crates/symbolicator-service/src/caching/tests.rs

```diff
@@ -27,12 +25,12 @@ fn tempdir() -> io::Result<tempfile::TempDir> {
     tempfile::tempdir_in(".")
 }
 
-fn write_file_and_metadata(path: &Path, contents: &[u8]) -> Result<()> {
+fn write_file_and_metadata(path: &Path, contents: &[u8], scope: Scope) -> Result<()> {
```

Comment:
Could already pass in the metadata

```suggestion
fn write_file_and_metadata(path: &Path, contents: &[u8], metadata: &Metadata) -> Result<()> {
```